Library/Homebrew/cmd/tests.rb

```diff
@@ -41,9 +41,14 @@ def tests
       args << "--trace" if ARGV.include? "--trace"
       if ARGV.value("only")
         ENV["HOMEBREW_TESTS_ONLY"] = "1"
-        test_name, test_method = ARGV.value("only").split("/", 2)
-        args << "TEST=test_#{test_name}.rb"
-        args << "TESTOPTS=--name=test_#{test_method}" if test_method
+
+        pairs = ARGV.value("only").split
```

Comment:
It will work if you make the thing after `--only=` a single argument as seen from the shell, e.g. by using comma (`,`) as a delimiter, or quoting the entire argument that includes the space like `--only="integration_cmds cmd_audit"` or `--only=integration_cmds\ cmd_audit`.